python-sdk/tests/sql/test_table.py

```diff
@@ -151,3 +152,76 @@ def test_if_table_object_can_be_pickled():
     """Verify if we can pickle Table object"""
     table = Table()
     assert pickle.loads(pickle.dumps(table)) == table
+
+
+@pytest.mark.parametrize(
```

Comment:
I think it would be good to add an integration test as well to see if extractor is getting called when operator is called

Reply:
added, please let me know how it is looking now